Integration tests fail for python 3.9 and 3.10

The codev integration tests fail for python 3.9 and 3.10, with the message

`FAILED tests/test_vector_parameter.py::TestVectorParameter::test_vector_parameter_like - TypeError: arrays to stack must be passed as a "sequence" type such as list or tuple.`

This is due to the line

`xs = np.hstack(p.sample() for p in pta.params)`

in `test_vector_parameters_like`.

PR #352 fixes this issue. Until merged, no new PRs will pass the integration tests.

--- a/tests/test_vector_parameter.py
+++ b/tests/test_vector_parameter.py
@@ -75,7 +75,7 @@ class TestVectorParameter(unittest.TestCase):
         pta = signal_base.PTA([s(self.psr)])
 
         # parameters
-        xs = np.hstack(p.sample() for p in pta.params)
+        xs = np.hstack([p.sample() for p in pta.params])
         params = {"B1855+09_red_noise_log10_rho": xs[1:], "B1855+09_efac": xs[0]}
 
         # test log likelihood

